providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/common/ChannelBuilder.java

```diff
@@ -0,0 +1,71 @@
+package dev.openfeature.contrib.providers.flagd.resolver.common;
+
+import dev.openfeature.contrib.providers.flagd.FlagdOptions;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.grpc.ManagedChannel;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.channel.epoll.Epoll;
+import io.netty.channel.epoll.EpollDomainSocketChannel;
+import io.netty.channel.epoll.EpollEventLoopGroup;
+import io.netty.channel.unix.DomainSocketAddress;
+import io.netty.handler.ssl.SslContextBuilder;
+
+import javax.net.ssl.SSLException;
+import java.io.File;
+
+/**
+ * gRPC channel builder helper.
+ */
+public class ChannelBuilder {
```

Comment:
Does this mean we can (hypothetically) use unix sockets for the sync connection too?

Reply:
yeah, this is possible. I reused our channel builder so that we have the option if needed.